SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -20,25 +21,26 @@ SpatialLoadBalanceEnforcer::SpatialLoadBalanceEnforcer(const PhysicalWorkerName&
 	check(InVirtualWorkerTranslator != nullptr);
 }
 
-void SpatialLoadBalanceEnforcer::OnAuthorityIntentComponentUpdated(const Worker_ComponentUpdateOp& Op)
+void SpatialLoadBalanceEnforcer::OnLoadBalancingComponentAdded(const Worker_AddComponentOp& Op)
 {
-	check(Op.update.component_id == SpatialConstants::AUTHORITY_INTENT_COMPONENT_ID);
+	// Should only be passed auth intent, ACL or component presence.
```

Comment:
We should probably only put the specifics in the HandlesComponent function. Otherwise these are a pain to keep in date.